### PR TITLE
Adds Switchmate documentation

### PR DESCRIPTION
**Pull request by Danielhiversen**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15535

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/switch.switchmate.markdown [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+To enable it, add the following lines to your `configuration.yaml`:
+
+```yaml
+switch:
+  - platform: switchmate
+    mac: 'cb:25:0b......'
+    name: "Name of the device"
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/switch.switchmate.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/switch.switchmate.markdown [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+This `Switchmate` switch platform allow you to control Switchmate [devices]( https://www.mysimplysmarthome.com/products/switchmate-switches/).
+
+To enable it, add the following lines to your `configuration.yaml`:
+
+```yaml
+switch:
+  - platform: switchmate
+    mac: 'cb:25:0b......'
+    name: "Name of the device"
```
Comment (frenck) [inline source/_components/switch.switchmate.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]
@Danielhiversen Thank you for the PR. I've left you some comments, could you please a look? 👍

File: source/_components/switch.switchmate.markdown [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  - platform: switchmate
+    mac: 'cb:25:0b......'
+```
+
+{% configuration %}
+mac:
+  description: Device MAC address.
+  required: true
+  type: string
+ name:
```
Comment (tjorim) [inline source/_components/switch.switchmate.markdown]: This indentation needs fixing

**Review by tjorim — CHANGES_REQUESTED** [review]

File: source/_components/switch.switchmate.markdown [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Switchmate"
+description: "Instructions on how to have Switchmate switches."
+date: 2018-08-17 22:41
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: switchmate.png
```
Comment (MartinHjelmare) [inline source/_components/switch.switchmate.markdown]: The name doesn't match the uploaded image.

File: source/_components/switch.switchmate.markdown [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+ha_category: Switch
+ha_release: 0.77
```
Comment (MartinHjelmare) [inline source/_components/switch.switchmate.markdown]: 0.78

File: source/_components/switch.switchmate.markdown [hunk 6]
```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Switchmate"
+description: "Instructions on how to have Switchmate switches."
```
Comment (MartinHjelmare) [inline source/_components/switch.switchmate.markdown]: Maybe replace "have" with "set up" or something similar?

File: source/_components/switch.switchmate.markdown (hunk 4 above)
Comment (Danielhiversen) [reply source/_components/switch.switchmate.markdown]: Opps

**Review by MartinHjelmare — DISMISSED** [review]

**Comment by MartinHjelmare** [comment]
@frenck ok?

**Comment by frenck** [comment]
@MartinHjelmare Little backed up because of my 2 weeks vacation. Assigned this to myself and will take a look shortly.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
✅ Approved. Can be merged as soon as the build passes.

**Pull request merged**